.github/workflows/gh-pages-deployment.yml

```diff
@@ -23,6 +22,13 @@ jobs:
     - name: Install Dependencies
       run: npm install
 
+    - name: Add subdomain
```

Comment:
subdomain is part of the domain name. 

for `ai-cfia.github.io`, the subdomain is: `ai-cfia`

I think you mean path. 

have you tried `/nachet-frontend` as the homepage? this is a relative path to the root of the server.

please review :

https://developer.mozilla.org/en-US/docs/Learn/Common_questions/Web_mechanics/What_is_a_URL

![image](https://github.com/ai-cfia/nachet-frontend/assets/538542/375ee96b-1f38-43f5-b689-b646cb6d3df8)

Reply:
got it